### PR TITLE
Move schema_editor to go/cmd/vtctld

### PR DESCRIPTION
**Pull request by yaoshengzhe**
@sougou @alainjobart 

Also removed the go/cmd/vtctld/templates/index.html since schema_editor covers all these stuff. Later, we can make the dashboard look better by moving existing UI components into this framework.

### CODE REVIEW

**Comment by yaoshengzhe** [comment]
one can verify the changes by running "test/demo.py"

File: go/cmd/vtctld/vtctld.go [hunk 1]
```diff
@@ -269,6 +257,11 @@ func main() {
 	})
 
 	// vschema editor
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
```
Comment (sougou) [inline go/cmd/vtctld/vtctld.go]: Do we want schema editor to be the landing page?

File: test/utils.py [hunk 2]
```diff
@@ -694,7 +694,7 @@ def start(self):
     args = environment.binary_args('vtctld') + [
             '-debug',
             '-templates', environment.vttop + '/go/cmd/vtctld/templates',
-            '-schema-editor-dir', environment.vttop + '/go/vt/vtgate',
+            '-schema-editor-dir', environment.vttop + '/go/cmd/vtctld',
```
Comment (sougou) [inline test/utils.py]: This should be obsolete, right?

File: go/cmd/vtctld/vtctld.go (hunk 1 above)
Comment (yaoshengzhe) [inline go/cmd/vtctld/vtctld.go]: Yes, I would image the vtctld page will be a single angular app. There will be always a navigation bar on the top, each tab renders page in a different way but they all live under host:port/#...

File: test/utils.py (hunk 2 above)
Comment (yaoshengzhe) [inline test/utils.py]: Correct, in fact, another coming pull request will merge schema_editor dir and templates dir. In other words, templates dir will contain all UI related stuff.

**Comment by sougou** [comment]
lgtm

**Pull request merged**